python/rpdk/java/templates/init/guided_aws/StubCreateHandler.java

```diff
@@ -0,0 +1,176 @@
+             */
+        } catch (final AwsServiceException e) {
+            throw new CfnGeneralServiceException(ResourceModel.TYPE_NAME, e);
+        }
+
+        logger.log(String.format("%s successfully created.", ResourceModel.TYPE_NAME));
+        return awsResponse;
+    }
+
+    /**
+     * If your resource requires some form of stabilization (e.g. service does not provide strong consistency), you will need to ensure that your code
+     * accounts for any potential issues, so that a subsequent read/update requests will not cause any conflicts (e.g. NotFoundException/InvalidRequestException)
+     * for more information -> https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-test-contract.html#resource-type-test-contract-communication
+     * @param awsRequest the aws service request to create a resource
+     * @param awsResponse the aws service response to create a resource
+     * @param proxyClient the aws service client to make the call
+     * @param model resource model
+     * @param callbackContext callback context
+     * @return boolean state of stabilized or not
+     */
+    private boolean stabilizedOnCreate(
+        final AwsRequest awsRequest,
+        final AwsResponse awsResponse,
+        final ProxyClient<SdkClient> proxyClient,
+        final ResourceModel model,
+        final CallbackContext callbackContext) {
+
+        // TODO: put your stabilization code here
+
+        final boolean stabilized = true;
+        logger.log(String.format("%s [%s] has been stabilized.", ResourceModel.TYPE_NAME, model.getPrimaryIdentifier()));
+        return stabilized;
+    }
+
+    /**
+     * If your resource is provisioned through multiple API calls, you will need to apply each subsequent update
+     * step in a discrete call/stabilize chain to ensure the entire resource is provisioned as intended.
+     * @param progressEvent of the previous state indicating success, in progress with delay callback or failed state
+     * @param proxyClient the aws service client to make the call
+     * @return progressEvent indicating success, in progress with delay callback or failed state
+     */
+    private ProgressEvent<ResourceModel, CallbackContext> postCreate(
+        final ProgressEvent<ResourceModel, CallbackContext> progressEvent,
+        final ProxyClient<SdkClient> proxyClient) {
+
+        final ResourceModel model = progressEvent.getResourceModel();
+        final CallbackContext callbackContext = progressEvent.getCallbackContext();
+        try {
+            // TODO: put your post creation resource update code here
+            // e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/commit/2077c92299aeb9a68ae8f4418b5e932b12a8b186#diff-c959c290785791b96fa4442573750fcdR69-R74
```

Comment:
again, we're asking them explicitly to use a call chain pattern and then saying "here's an example of what I'm talking about" and that example doesn't use the call chain pattern...

Reply:
removed